src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import java.text.ParseException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Dealine Class to represent a class that will store Deadline object that extends from Tasks Class
+ * @author amresh A0235398R
+ */
+public class Deadline extends Task {
+    protected String by;
+    protected LocalDate date;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    /**
+     * Method to convert deadline String to LocalDate format
+     * @return boolean true value if converted
```

Comment:
Should there be an empty line between the description and return sections for readability? I noticed this in several of your javadocs, here's the [reference material](https://se-education.org/guides/conventions/java/intermediate.html).